template_workflows/dependabot.yml

```
@@ -8,7 +8,7 @@ updates:
 
   # We instruct dependabot to open PRs if a newer version
   # of a github action has been released.
-  - package-ecosystem: "github-actions"
+  - package-ecosystem: 'github-actions'
```

:cherries: for consistency w/ double quotes almost everywhere else in the file:
```suggestion
  - package-ecosystem: "github-actions"
```